freetextresponse/freetextresponse.py

```diff
@@ -18,11 +18,69 @@
 from xblock.fragment import Fragment
 from xblock.validation import ValidationMessage
 from xblockutils.studio_editable import StudioEditableXBlockMixin
+from xblockutils.publish_event import PublishEventMixin
```

Comment:
need to remove

Reply:
remember to remove all the instances of use below in this file.